sale_brand/models/sale_order.py

```diff
@@ -16,3 +16,7 @@ def _prepare_invoice(self):
                 'brand_id': order.brand_id.id,
             })
         return invoice_vals
+
+    @api.onchange('team_id')
+    def _onchange_team_id(self):
+        self.brand_id = self.team_id.brand_id
```

Comment:
Would it make sense to do this only if team_id.brand_id is set?

Reply:
I have concerns with that behavior:
- I pick a Branded Sales Team, Brand X is set
- The I pick an Unbranded Sales Team, the brand is not cleared, and it stays Brand X
- The Unbranded Team SO will print with the Brand X template, and that will be an defect to the Sales Manager